quipucords/api/source/serializer.py

```diff
@@ -197,6 +210,7 @@ def create(self, validated_data):
             source.options = options
 
         source.hosts = json.dumps(hosts_list)
+        source.exclude_hosts = json.dumps(exclude_hosts_list)
```

Comment:
Add a check to make sure exclude_hosts isn't None before we assign it to the source